include/avif/internal.h

```diff
@@ -431,14 +431,20 @@ typedef size_t avifBoxMarker;
 
 typedef struct avifBoxHeader
 {
+    // Size of the box in bytes, excluding the number of bytes read to know the size and the type.
```

Comment:
Optional: "the number of bytes read to know the size and the type" is the size of the box header, right? It seems that "excluding the size of the box header" is clear enough.